### ISSUE TITLE
Why does "Topics and Themes" have more options in the modal facet options box

### ISSUE THREAD

**Issue by crarugal** [ukwa/ukwa-ui#164]
For example, searching for "http" so that many results are returned for testing, I noticed that the modal box offers many more options for topics and themes than are available on the topics and themes page.
 It seems there are multiple options for filtering by NHS collections, which don't seem to be special collections on ACT.

![image](https://user-images.githubusercontent.com/18530934/46207537-a685f700-c31f-11e8-8d7f-8ba710cd7738.png)

The are some of the NHS options available:

Health and Social Care Act 2012 - NHS Reforms\|NHS\|Gateways
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Cancer Networks
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|North of England SHA Cluster\|North West\|Greater Manchester
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|North of England SHA Cluster\|Yorkshire and the Humber\|South    Yorkshire and Bassetlaw
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Specialised Commissioning    Group
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Healthwatch
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Press Comment
Housing and Social Housing
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|South of England SHA Cluster\|South Central\|Southampton,    Hampshire, Isle of Wight & Portsmouth
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|South of England SHA Cluster\|South West\|Bournemouth, Poole and    Dorset
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|South of England SHA Cluster\|South West\|Bristol, North Somerset    and South Gloucestershire
Zines and Fanzines
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|North of England SHA Cluster\|North West\|Cheshire, Warrington,    Wirral
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|South of England SHA Cluster\|South Central\|Berkshire
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|Midlands and East SHA Cluster\|West Midlands\|Birmingham and    Solihull
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|South of England SHA Cluster\|South East Coast\|Sussex
Health and Social Care Act 2012 - NHS Reforms\|NHS\|NHS programmes
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|Midlands and East SHA Cluster\|East Midlands\|East Midlands
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|Midlands and East SHA Cluster\|East of England\|Cambridgeshire and    Peterborough
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|South of England SHA Cluster\|South West\|Gloucestershire and    Swindon
Magic
19th Century English Literature
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|Midlands and East SHA Cluster\|West Midlands\|West Mercia
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|South of England SHA Cluster\|South East Coast\|Kent and    Medway
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|North of England SHA Cluster\|North East\|Tees
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|North of England SHA Cluster\|North West\|Lancashire
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|North of England SHA Cluster\|Yorkshire and the    Humber\|Humber
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|Midlands and East SHA Cluster\|West Midlands\|Black Country
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|Midlands and East SHA Cluster\|West Midlands\|Staffordshire
Health and Social Care Act 2012 - NHS Reforms\|NHS\|Strategic Health    Authorities\|North of England SHA Cluster\|Yorkshire and the    Humber\|Leeds

**Comment by crarugal** [comment]
It looks as though sub collections from the parent collection "Health and Social Care Act 2012 - NHS Reforms " are being pulled in

![image](https://user-images.githubusercontent.com/18530934/46208021-42643280-c321-11e8-9ad4-0edf0d8d1a77.png)

**Comment by min2ha** [comment]
Could be related to 
https://github.com/ukwa/ukwa-ui/issues/153
as data synchronization is need

**Comment by jasonwebber-bl** [comment]
I agree that ideally sub collections shouldn't be immediately included. I think there is a technical reason why this is @nicolabingham might know?

**Comment by crarugal** [comment]
As an example, searching for "http" will return these options in the facet modal box; below is a comparison of what topics and themes are in Alpha, what's returned in the facet options modal box, and what special collections are in ACT:
 
[comparing  special collections and topics and themes.xlsx](https://github.com/ukwa/ukwa-ui/files/2428568/comparing.special.collections.and.topics.and.themes.xlsx)
